ci/build.sh

```diff
@@ -4,12 +4,55 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/.."
 
+if [[ -z ${BUILDKITE+x} ]]; then
+  echo "This script is intended for internal Improbable use."
+  exit 0
+fi
+
+SPATIALOS_TOOLBELT_VERSION="latest"
 PLATFORM=""
+BINARY=""
```

Comment:
This variable should be called `SPATIAL_BINARY` or similar.

Reply:
Renamed to `SPATIAL_BINARY`